content/en/monitors/service_level_objectives/metric.md

```diff
@@ -10,12 +12,12 @@ further_reading:
 
 ## Overview
 
-Event-based SLOs are useful for a count-based stream of data where you are differentiating good and bad events. 
-There is one type of event-based SLO - a metric query SLO. A metric query uses the sum of the good events divided by the sum of total events over time to calculate a Service Level Indicator (or SLI).
+Metric-based SLOs are useful for a count-based stream of data where you are differentiating good and bad events. 
+There is one type of metric-based SLO - a metric query SLO. A metric query uses the sum of the good events divided by the sum of total events over time to calculate a Service Level Indicator (or SLI).
 
 ## Setup
 
-On the [SLO status page][1], select **New SLO +**. Then select [**Event**][2].
+On the [SLO status page][1], select **New SLO +**. Then select [**Metric**][2].
 
 ### Configuration
 
```

Comment:
Can comment on it, but the last line in the file is `[2]: https://app.datadoghq.com/slo/new/event`, I assume we'll update that link and redirect, but it should probably be `[2]: https://app.datadoghq.com/slo/new/metric`

Reply:
nice catch